v2/datastream-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToBigQuery.java

```diff
@@ -181,8 +178,7 @@ public interface Options
         optional = true,
         description = "Name or template for the stream to poll for schema information.",
         helpText =
-            "This is the name or template for the stream to poll for schema information. Default is"
-                + " {_metadata_stream}. The default value is enough under most conditions.")
+            "This is the name or template for the stream to poll for schema information. Default is {_metadata_stream}. The default value is enough under most conditions.")
```

Comment:
```suggestion
            "The name or the template for the stream to poll for schema information. Defaults to: {_metadata_stream}. The default value is usually enough.")
```

Reply:
Does the default value actually have curly braces, or are those supposed to indicate a variable? If the curly braces aren't part of the default value, we should remove them.